ref-exchange/src/errors.rs

```diff
@@ -0,0 +1,12 @@
+// Storage errors //
+
+pub const ERR10_ACC_NOT_REGISTERED: &str = "E10: account not registered";
+pub const ERR11_INSUFFICIENT_STORAGE: &str = "E11: insufficient $NEAR storage deposit";
+pub const ERR12_TOKEN_NOT_WHITELISTED: &str = "E12: token not whitelisted";
+
+// Account Deposits //
+
+pub const ERR21_TOKEN_NOT_REG: &str = "E21: token not registered";
+pub const ERR22_NOT_ENOUGH_TOKENS: &str = "E23: not enough tokens in deposit";
+// pub const ERR23_NOT_ENOUGH_NEAR: &str = "E23: not enough NEAR in deposit";
```

Comment:
let's avoid checking in commented code to keep code clean

Reply:
I will use it in the next PR.